models/domain_tlds.php

```diff
@@ -6,7 +6,7 @@
  *
  * @link https://www.blesta.com Blesta
  */
-class DomainManagerTlds extends DomainManagerModel
+class DomainTlds extends DomainsModel
```

Comment:
Plugin models usually starts with the name of the plugin, in this case I think it should be DomainsTlds

Reply:
Or if you want to keep "DomainTlds", maybe "DomainsDomainTlds", similar to the OrderOrders model.

Reply:
Yea, I'll update it.

Reply:
Updated